### PR TITLE
Default security token

### PR DESCRIPTION
**Pull request by jujugrrr**
Hello,
The ec2 tools are able to use the security token and the IAM profile, however it doesn't work if the access keys are defined as an ENV variable

### CODE REVIEW

**Comment by jujugrrr** [comment]
I've not updated the tests

File: recipes/api.rb [hunk 1]
```diff
@@ -10,6 +10,6 @@
   source 'aws_keys.sh.erb'
   owner 'root'
   group 'root'
-
   mode 0755
+  only_if { node['aws_developer_tools']['deploy_key?'] }
```
Comment (anujbiyani) [inline recipes/api.rb:14]: Looks like there's a slight typo here; the attribute is defined above (https://github.com/Lytro/aws_developer_tools/pull/5/files#L0R3) as `deploy_keys?` but here it's written as `deploy_key?`

File: attributes/default.rb [hunk 2]
```diff
@@ -1,11 +1,12 @@
 default['aws_developer_tools']['install_java?'] = true
 default['aws_developer_tools']['install_ruby?'] = true
+default['aws_developer_tools']['deploy_keys?']  = false
 
 default['aws_developer_tools']['aws_access_key'] = 'Your Access Key'
 default['aws_developer_tools']['aws_secret_key'] = 'Your Secret Key'
 
 ## EC2 Tools ##
-default['aws_developer_tools']['ec2_tools_target'] = '/usr/local/share/ec2_tools'
+default['aws_developer_tools']['ec2_tools_target'] = '/opt/aws/apitools/ec2'
```
Comment (anujbiyani) [inline attributes/default.rb]: Just curious; what motivated this change?

Comment (jujugrrr) [inline attributes/default.rb]: My bad it should not have been in the pull request, I've just used this path to match with the default path on the AWS Linux ami

File: recipes/api.rb (hunk 1 above)
Comment (jujugrrr) [inline recipes/api.rb:14]: Indeed

**Comment by anujbiyani** [comment]
@jujugrrr Thanks for the pull request! Once the typo is fixed and the default path is changed back I can merge this. Updating the tests shouldn't be necessary; the change is around an added `only_if` block, and ChefSpec can't test those anyway.

**Comment by anujbiyani** [comment]
Also in the future it'd be better if you create a topic branch for your changes and then submit a pull request from there; helps keep the changes isolated.

**Comment by jujugrrr** [comment]
You're welcome, I'll change it tomorrow ;-)

**Pull request closed without merging**